src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BQSRIntegrationTest.java

```diff
@@ -0,0 +1,60 @@
+package org.broadinstitute.hellbender.tools.walkers.bqsr;
+
+import htsjdk.samtools.ValidationStringency;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.Main;
+import org.broadinstitute.hellbender.tools.validation.CompareBaseQualities;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+//Note: has to be in this package to have access to Main.instanceMain
+public final class BQSRIntegrationTest extends CommandLineProgramTest {
```

Comment:
We already have a `BaseRecalibratorIntegrationTest`, and you've now created `BQSRIntegrationTest`??